docs/src/modules/utils/helpers.js

```diff
@@ -200,9 +200,11 @@ function getDependencies(raw, options = {}) {
  * @return {String}       The cookie value
  */
 function getCookie(name) {
-  const value = `; ${document.cookie}`;
-  const parts = value.split(`; ${name}=`);
-  if (parts.length === 2) return parts.pop().split(';').shift();
+  if (process.browser) {
+    const value = `; ${document.cookie}`;
+    const parts = value.split(`; ${name}=`);
+    if (parts.length === 2) return parts.pop().split(';').shift();
```

Comment:
That poor array is getting torn apart before it could live for some time :laughing: